src/watcher.js

```diff
@@ -266,6 +272,30 @@ class Watcher extends SafeEventEmitter {
         this.on('load.chat', updateChannel);
         this.on('load.vod', updateChannel);
     }
+
+    directoryObserver() {
+        const observe = (watcher, element) => {
+            if (!element) return;
+            if (watcher) watcher.disconnect();
+            watcher.observe(element, {childList: true, subtree: true});
+        };
+
+        directoryWatcher = new window.MutationObserver(mutations =>
+            mutations.forEach(mutation => {
+                for (const el of mutation.addedNodes) {
+                    const $el = $(el);
```

Comment:
added nodes only returns added nodes, what about initial load for the case where betterttv loads after the elements have rendered?

Reply:
I'm not sure. I had it just hide on load to fix that.